proto/proto/light_wallet.proto

```diff
@@ -11,16 +12,23 @@ service LightWallet {
   rpc CompactBlockRange(CompactBlockRangeRequest) returns (stream chain.CompactBlock);
   rpc ChainParams(ChainParamsRequest) returns (chain.ChainParams);
   rpc ValidatorInfo(ValidatorInfoRequest) returns (stream stake.ValidatorInfo);
+  rpc AssetList(AssetListRequest) returns (chain.KnownAssets);
+}
+
+// Lists all assets in Asset Registry
+message AssetListRequest {
+  // The expected chain id (empty string if no expectation).
+  string chain_id = 1;
 }
 
 // Requests a range of compact block data.
 message CompactBlockRangeRequest {
   // The expected chain id (empty string if no expectation).
-  string chain_id = 3;
+  string chain_id = 1;
```

Comment:
Since we're breaking the RPC anyways, I took the opportunity to reorder the fields so that `chain_id` came first.